src/fides/api/ops/schemas/saas/saas_config.py

```diff
@@ -203,13 +203,39 @@ def validate_override(cls, values: Dict[str, Any]) -> Dict[str, Any]:
         return values
 
 
+class SaaSRequestMap(BaseModel):
+    """A map of actions to SaaS requests"""
+
+    read: Union[SaaSRequest, List[SaaSRequest]] = []
```

Comment:
One of the biggest changes was moving away from the `Dict[Literal["read", "update", "delete"], SaaSRequest]` type hint for endpoint requests since the `reads` can be either a single `SaaSRequest` or a list of `SaaSRequests`. I went with the union type to avoid having to specify each read request as a list.